Sources/CustomDump/Conformances/UIKit.swift

```diff
@@ -60,5 +60,30 @@
         }
       }
     }
+    
+    @available(iOS 2, macCatalyst 13, tvOS 9, *)
+    @available(watchOS, unavailable)
+    extension UIControl.State: CustomDumpStringConvertible {
+      public var customDumpDescription: String {
+        switch self {
+        case .normal:
+          return "UIControl.State.normal"
+        case .highlighted:
+          return "UIControl.State.highlighted"
+        case .disabled:
+          return "UIControl.State.disabled"
+        case .selected:
+          return "UIControl.State.selected"
+        case .focused:
+          return "UIControl.State.focused"
+        case .application:
+          return "UIControl.State.application"
+        case .reserved:
+          return "UIControl.State.reserved"
+        default:
+          return "UIControl.State.(default, rawValue: \(self.rawValue))"
```

Comment:
Should this be `@unknown default:`, like the rest?

```suggestion
        @unknown default:
          return "UIControl.State.(@unknown default, rawValue: \(self.rawValue))"
```

Reply:
@stephencelis 
May be No, UIControl.State is struct.
What you see like case is static property.

Reply:
Ah right :) Thanks for the PR!